data/readreplica.py

```diff
@@ -133,5 +139,8 @@ def raw(cls, *args, **kwargs):
             query._database = cls._select_database()
         elif cls._in_readonly_mode():
             raise ReadOnlyModeException()
+        elif query._sql.lower().startswith("insert "):
+            if hasattr(cls, "__deprecated_model"):
```

Comment:
This is already checked above with `is_deprecated_model`

Reply:
Fixed